templates/zerver/api/update-message.md

```diff
@@ -52,6 +53,7 @@ print(client.update_message({
 }))
 
 ```
+
 </div>
```

Comment:
Again, these are rendered on our `/api` page and these are not really supposed to be for the GitHub Markdown viewer. I am not sure what "issue" you are trying to fix here. Could you please explain more? And make sure you run these locally and navigate to `/api` in your dev env and make sure that the changes matter there! :)

Reply:
Okay, so I see what you are trying to fix. Just split it into another commit and then make sure that it renders correctly locally on /api. :)

Reply:
@eeshangarg  I came across this doc on GitHub for one issue in `zulip-js`. So it was a quick fix :)
Now separated into two commits.